### PR TITLE
add opt-in to newsletter

### PR DESCRIPTION
**Pull request by laserchicken**
request: https://trello.com/c/EbZKffod/349-add-extra-opt-in-to-newsletter-per-legal

### CODE REVIEW

File: src/components/footer/footer.hbs [hunk 1]
```diff
@@ -21,7 +21,7 @@
           </p>
 
           <label for="newsletter-register-email">Email address</label>
-          <input class="newsletter-register__email" id="newsletter-register-email" name="sailthru[email]" type="email" placeholder="your@email.com" autocomplete="email" autocapitalize="none" autocorrect="off" required>
+          <input class="newsletter-register__email" id="newsletter-register-email" name="sailthru[email]" type="email" placeholder="your@email.com" autocomplete="email" autocapitalize="none" autocorrect="off" required="required">
```
Comment (thomasthesecond) [inline src/components/footer/footer.hbs]: Don't need `required="required"` here; `required` by itself is correct.

File: src/components/footer/footer.hbs [hunk 2]
```diff
@@ -31,7 +31,10 @@
             {{/each}}
           </select>
 
-          <button class="newsletter-register__button js-submit" data-lpa-action="newsletter" data-lpa-category="account">Sign up</button>
+          <input id="newsletter-register-optin" class="newsletter-register__optin js-newsletter-optin" type="checkbox" name="sailthru[legalOptIn]" required="required" value="true">
+          <div class="newsletter-register__text">Tick to opt-in. Opt out at any time via the 'unsubscribe' link in the footer of the emails.</div>
+
+          <input class="newsletter-register__button js-submit" data-lpa-action="newsletter" data-lpa-category="account" value="Sign up" type="submit" />
```
Comment (thomasthesecond) [inline src/components/footer/footer.hbs]: Is there a reason we're not using a `button` anymore?

File: src/components/footer/footer.hbs [hunk 3]
```diff
@@ -31,7 +31,10 @@
             {{/each}}
           </select>
 
-          <button class="newsletter-register__button js-submit" data-lpa-action="newsletter" data-lpa-category="account">Sign up</button>
+          <input id="newsletter-register-optin" class="newsletter-register__optin js-newsletter-optin" type="checkbox" name="sailthru[legalOptIn]" required="required" value="true">
```
Comment (thomasthesecond) [inline src/components/footer/footer.hbs]: `required`

**Comment by nickfranciosi** [comment]
@laserchicken I know that @katelynp43 has been making some similar updates in backpack. She got rid of [sailthru] https://github.com/lonelyplanet/backpack-ui/blob/master/src/components/newsletter/index.jsx#L199. Just wanted to point that out, but I am not fully aware of the fix, so it might not be an issue

**Comment by laserchicken** [comment]
@nickfranciosi yeah, she asked me for help with newsletter form in the footer; she is using new notation but `sailthru` is still supported (https://github.com/lonelyplanet/gonzo/blob/master/app/controllers/newsletter_controller.rb#L18) - should be ok, thanks

File: src/components/footer/footer.hbs (hunk 1 above)
Comment (katelynp43) [inline src/components/footer/footer.hbs]: Please update the variable from `sailthru[email]` to `newsletter[email]`

File: src/components/footer/footer.hbs [hunk 4]
```diff
@@ -21,7 +21,7 @@
           </p>
 
           <label for="newsletter-register-email">Email address</label>
-          <input class="newsletter-register__email" id="newsletter-register-email" name="sailthru[email]" type="email" placeholder="your@email.com" autocomplete="email" autocapitalize="none" autocorrect="off" required>
+          <input class="newsletter-register__email" id="newsletter-register-email" name="sailthru[email]" type="email" placeholder="your@email.com" autocomplete="email" autocapitalize="none" autocorrect="off" required="required">
 
           <label for="newsletter-register-country">Country</label>
           <select class="newsletter-register__country" id="newsletter-register-country" name="sailthru[country]">
```
Comment (katelynp43) [inline src/components/footer/footer.hbs]: Please update the variable from `sailthru[country]` to `newsletter[country]`

File: src/components/footer/footer.hbs (hunk 3 above)
Comment (katelynp43) [inline src/components/footer/footer.hbs]: Please update the variable from `sailthru[legalOptIn]` to `newsletter[legalOptIn]`

**Comment by laserchicken** [comment]
as I said above @katelynp43 `sailthru` is still supported, so no need to update anything here

**Comment by katelynp43** [comment]
@laserchicken I know it is still supported but there has been big effort to remove sailthru variables as much as possible so I would like to remove them and leave the code cleaner than we found it. :)

**Comment by laserchicken** [comment]
updated @katelynp43

**Comment by laserchicken** [comment]
@thomasthesecond if I use `button` it will not validate the form - you can submit empty inputs

**Comment by thomasthesecond** [comment]
You can add type="submit" on the `button`

**Pull request merged**